modules/op-release-notes-1-11.adoc

```diff
@@ -202,4 +202,40 @@ With this command, you can also specify a custom GitHub API URL by using the `--
+
+* Before this update, a shell script in a task had to be run as root. After this update, the shell scripts image has the non-root user ID set, so a task that includes a shell script, such as the `git-clone` task, can be run as a non-root user within the pod.
+
+* Before this update, in {pipelines-title} 1.11.0, when a pipeline run was included in a Tekton bundle, the `kind` value was sometimes wrongly set to `Task`. The issue did not exist in previous versions of {pipelines-title}. After this update, the `kind` value is set correctly.
+
+* Before this update, the `disable-ha` flag was not correctly passed to the `tekton-pipelines` controller, so the High Availability (HA) features of {pipelines-title} could not be enabled. After this update, the `disable-ha` flag  is correctly passed and you can enable HA features as required.
+
+* Before this update, you could not set the URL for {tekton-hub} and {artifact-hub} for the hub resolver, so you could use only the preset addresses of {tekton-hub} and Partifact-hub}. After this update, you can configure the URL for {tekton-hub} and {artifact-hub} for the hub resolver, for example, to use a custom {tekton-hub} instance that you installed.
+
+* After this update, the SHA digest of the `git-init` image corresponds to version 1.10.5, which is the latest released version of the image.
+
+* Before this update, the `tekton-pipelines-controller` leader election used a config map named `config-leader-election`. This name is the default value for knative controllers, so the configuration process for {pipelines-shortname} could affect other controllers and vice versa. After this update, the leader election uses a unique config name, so the configuration process for {pipelines-shortname} does not affect other controllers and is not affected by other controllers.
+
+* Before this update, when a user who has no write access to a GitHub repository opened a pull request, {pac} CI/CD actions would show as skipped in GitHub. After this update, {pac} CI/CD actions are executed correctly in this case.
+
+* Before this update, {pac} ran CI/CD actions for every pull request into a branch that matched a configured branch name. After this update, {pac} runs CI/CD actions only when the source branch of the pull request matches the exact configured branch name.
+
+* Before this update, a `tkn pac resolve -f` command with a directory specified did not request a GitHub API token even when `{{ git_auth_secret }}` was present in the directory. After this update, the `tkn pac resolve -f` command requests a GitHub API token in this case.
+
+* Before this update, the `tkn pac resolve -f` command failed if a cluster was running an older version of {pipelines-shortname}, which did not yet support the `v1` API. After this update, you can specify the `--v1beta` or `-B` flag in this command dto use the `v1beta` API.
```

Comment:
If i remember correctly i think we are not shipping cli in 1.11.1 so if thats the case we need to remove these release notes
cc @concaf, [shverma@redhat.com](mailto:shverma@redhat.com)